webscrapbook/cli.py

```diff
@@ -282,6 +299,28 @@ def main():
         help="""generate more assistant files. (with --book)""")
     parser_config.add_argument('-e', '--edit', default=False, action='store_true',
         help="""edit the config file. (with --book or --user)""")
+    
+    # subcommand: scrapbook
+    parser_scrapbook = subparsers.add_parser('scrapbook', aliases=['scrap'],
+        help=cmd_scrapbook.__doc__, description=cmd_scrapbook.__doc__)
+    parser_scrapbook.set_defaults(func=cmd_scrapbook)
+    parser_scrapbook.add_argument('directory',
+        help="""scrapbook directory""")
+    subparser_scrapbook = parser_scrapbook.add_subparsers(metavar='SCRAPBOOK_COMMAND',
+        help="""The scrapbook command to run. Add --help(-h) after the command for usage details.""")
+    parser_sort = subparser_scrapbook.add_parser('sort')
+    parser_sort.set_defaults(func=cmd_sort)
+    parser_sort.add_argument('folder',
+        help="""id for folder to sort""")
+    parser_sort.add_argument('key',
+        choices=['title','create', 'modify', 'source', 'comment'],
+        help=""""metadata key to sort on""")
```

Comment:
Id should be added in the choice. And maybe provide a default value?

Reply:
I will add the default value `create`. And yes I didn't even think to add sorting on multiple keys or sorting on Id, so I will add that!

Reply:
This may be somehow controversial. But I think using id as the default key is more reasonable, although not really more natural or useful.

Reply:
I actually might agree, the id is implicitly in order or creation so I will change it.

Reply:
Okay everything here is done: default key is id, id sort key added, can now sort on multiple keys. Since it is now possible to sort on multiple keys I had to change the argument to `-k create id`